src/V3ParseSym.h

```diff
@@ -130,6 +130,13 @@ class V3ParseSym final {
         UASSERT_OBJ(!m_sympStack.empty(), nodep, "symbol stack underflow");
         m_symCurrentp = m_sympStack.back();
     }
+    AstNodeModule* findTopNodeModule(bool requireNoneNull = true) {
+        for (VSymEnt* const symp : vlstd::reverse_view(m_sympStack)) {
+            if (AstNodeModule* const modp = VN_CAST(symp->nodep(), NodeModule)) return modp;
+        }
+        if (requireNoneNull) UASSERT(false, "fail to find current module");
+        return nullptr;
+    }
```

Comment:
Though unlikely to fire, to help users we try to always associate assertions with the source causing them.

```suggestion
    AstNodeModule* findTopNodeModule(FileLine* fl, bool requireNoneNull = true) {
        for (VSymEnt* const symp : vlstd::reverse_view(m_sympStack)) {
            if (AstNodeModule* const modp = VN_CAST(symp->nodep(), NodeModule)) return modp;
        }
        if (requireNoneNull) fl->v3fatalSrc("fail to find current module");
        return nullptr;
    }
```